Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCheckPointFirewall.yaml

```diff
@@ -0,0 +1,260 @@
+      , "38","IDPR-CMTP"
+      , "39","TP++"
+      , "40","IL"
+      , "41","IPv6"
+      , "42","SDRP"
+      , "43","IPv6-Route"
+      , "44","IPv6-Frag"
+      , "45","IDRP"
+      , "46","RSVP"
+      , "47","GRE"
+      , "48","DSR"
+      , "49","BNA"
+      , "50","ESP"
+      , "51","AH"
+      , "52","I-NLSP"
+      , "53","SWIPE (deprecated)"
+      , "54","NARP"
+      , "55","MOBILE"
+      , "56","TLSP"
+      , "57","SKIP"
+      , "58","IPv6-ICMP"
+      , "59","IPv6-NoNxt"
+      , "60","IPv6-Opts"
+      , "61",""
+      , "62","CFTP"
+      , "63",""
+      , "64","SAT-EXPAK"
+      , "65","KRYPTOLAN"
+      , "66","RVD"
+      , "67","IPPC"
+      , "68",""
+      , "69","SAT-MON"
+      , "70","VISA"
+      , "71","IPCV"
+      , "72","CPNX"
+      , "73","CPHB"
+      , "74","WSN"
+      , "75","PVP"
+      , "76","BR-SAT-MON"
+      , "77","SUN-ND"
+      , "78","WB-MON"
+      , "79","WB-EXPAK"
+      , "80","ISO-IP"
+      , "81","VMTP"
+      , "82","SECURE-VMTP"
+      , "83","VINES"
+      , "84","TTP"
+      , "84","IPTM"
+      , "85","NSFNET-IGP"
+      , "86","DGP"
+      , "87","TCF"
+      , "88","EIGRP"
+      , "89","OSPFIGP"
+      , "90","Sprite-RPC"
+      , "91","LARP"
+      , "92","MTP"
+      , "93","AX.25"
+      , "94","IPIP"
+      , "95","MICP (deprecated)"
+      , "96","SCC-SP"
+      , "97","ETHERIP"
+      , "98","ENCAP"
+      , "99",""
+      , "100","GMTP"
+      , "101","IFMP"
+      , "102","PNNI"
+      , "103","PIM"
+      , "104","ARIS"
+      , "105","SCPS"
+      , "106","QNX"
+      , "107","A/N"
+      , "108","IPComp"
+      , "109","SNP"
+      , "110","Compaq-Peer"
+      , "111","IPX-in-IP"
+      , "112","VRRP"
+      , "113","PGM"
+      , "114",""
+      , "115","L2TP"
+      , "116","DDX"
+      , "117","IATP"
+      , "118","STP"
+      , "119","SRP"
+      , "120","UTI"
+      , "121","SMP"
+      , "122","SM (deprecated)"
+      , "123","PTP"
+      , "124","ISIS over IPv4"
+      , "125","FIRE"
+      , "126","CRTP"
+      , "127","CRUDP"
+      , "128","SSCOPMCE"
+      , "129","IPLT"
+      , "130","SPS"
+      , "131","PIPE"
+      , "132","SCTP"
+      , "133","FC"
+      , "134","RSVP-E2E-IGNORE"
+      , "135","Mobility Header"
+      , "136","UDPLite"
+      , "137","MPLS-in-IP"
+      , "138","manet"
+      , "139","HIP"
+      , "140","Shim6"
+      , "141","WESP"
+      , "142","ROHC"
+      , "143","Ethernet"
+      , "253",""
+      , "254",""
+      , "255","Reserved"];
+  let DirectionLookup=datatable(CommunicationDirection:string,NetworkDirection:string)
+  [
+      "Incoming","Inbound", 
+      "Outgoing","Outbound", 
+      "Internal","Local"];
+  let ActionLookup=datatable(DvcOriginalAction:string,DvcAction:string)
+  [
+      "Accept","Allow", 
+      "Allow","Allow", 
+      "Drop","Drop", 
+      "Reject","Deny", 
+      "Encrypt","Encrypt", 
+      "Decrypt","Decrypt",
+      "Bypass","Allow", 
+      "Block","Deny", 
+      "Detect","Deny", 
```

Comment:
I don't know if Detect->Deny. Not sure how to check that. I suspect that all the values below may not belong in a NetworkSession. Did you see them in events?

Reply:
No, I can only see Accept and Deny values. Removing all after Block for now.